src/components/App.js

```diff
@@ -29,17 +30,29 @@ class App extends Component {
     })
   }
 
+  componentDidMount = () => {
+    fetch(endpoints.movies)
+      .then(response => response.json())
+      .then(data => this.setState({
+        moviePosters: data.movies
+      }))
+      .catch(err => this.setState({
+        error: err.message
+      }))
+  }
+
```

Comment:
I have seen some people place their fetch calls into a separate file.  Personally, I think it might be overkill in a project of this size, but it is interesting to think about how one might implement it!